include/symbolizer.hpp

```diff
@@ -103,24 +103,34 @@ class Symbolizer {
   };
 
   struct BlazeSymbolizerWrapper {
-    static blaze_symbolizer_opts create_opts(bool inlined_fns) {
+    static blaze_symbolizer_opts create_opts_with_debug(bool inlined_fns) {
       return blaze_symbolizer_opts{.type_size = sizeof(blaze_symbolizer_opts),
                                    .auto_reload = false,
                                    .code_info = true,
                                    .inlined_fns = inlined_fns,
                                    .demangle = false,
                                    .reserved = {}};
     }
-
-    explicit BlazeSymbolizerWrapper(std::string elf_src, bool inlined_fns)
-        : opts(create_opts(inlined_fns)),
+    static blaze_symbolizer_opts create_opts_no_debug() {
```

Comment:
I don't think you need to change `blaze_symbolizer_opts`, calling `blaze_symbolize_elf_virt_offsets` with `blaze_symbolize_src_elf.debug_syms` set to false should be enough.